packages/modules/order/src/utils/actions/return-item.ts

```diff
@@ -57,15 +57,15 @@ OrderChangeProcessing.registerActionType(ChangeActionType.RETURN_ITEM, {
     }
 
     const quantityAvailable = MathBN.sub(
-      existing!.detail?.shipped_quantity ?? 0,
+      existing!.detail?.fulfilled_quantity ?? 0,
```

Comment:
comment: Not to be addressed now, but I think this will need to change by introducing a `delivered_quantity`. At this point, you shouldn't even be able to request a return, because the items are still at the warehouse. Canceling should suffice. I created a ticket for it for now.

Reply:
I think this validation of if the item was shipped should be outside the module, in the workflow. As this involves other module.
That is why I just changed that to check the fulfilled_quantity, which is the bare minimum for the module to support